<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalizationStringManagerClass</class>
 <widget class="QMainWindow" name="LocalizationStringManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>297</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>字符串本地化管理器</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="pbtn_openFile">
         <property name="text">
          <string>打开文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_filePath">
         <property name="text">
          <string>文件路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ledit_filePath">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame_search">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,1,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>查询</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_charId">
            <property name="text">
             <string>字符串id：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledit_search_id"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>地区或者语言：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cBox_search_local"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_search">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>查询结果：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledit_search_result"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_store">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,1,1,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>存储</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>字符串id：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledit_store_id"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>地区或者语言：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cBox_store_local"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="ptbn_store">
          <property name="text">
           <string>存储</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>存储结果：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledit_store_result"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../qrc/LocalizationStringManager.qrc"/>
 </resources>
 <connections/>
</ui>
